README.md

```diff
@@ -4,6 +4,8 @@
 
 This documentation is oriented towards developers, if you'd like to learn more about our data and how to access it please refer to [Toilet Map Explorer](https://www.toiletmap.org.uk/explorer).
 
+![Powered by Vercel](./public/powered-by-vercel.svg)
```

Comment:
Should this be a link back to vercel too?

Reply:
agreed, that would make sense